cmssw-pr-test-config

```diff
@@ -1,2 +1,3 @@
 PR_TEST_MATRIX_EXTRAS=1306.0,101.0,9.0,25202.0,10224.0,250202.181
 PR_TEST_MATRIX_EXTRAS_GPU=10824.501,10824.502,10824.511,10824.512
+PR_TEST_MATRIX_EXTRAS_PROFILING=23434.21
```

Comment:
can you read this file to get the workflow number for profiling job?

Reply:
Line 962 in pr_testing/test_multiple_prs.sh should read this,

Reply:
No, I am taking about https://cmssdt.cern.ch/jenkins/job/ib-run-profiling/configure job to use this workflow (instead of hard coding in the jenkins job)

Reply:
OK, I will update schedule-additional-tests to read the workflow number from  cmssw-pr-test-config.